server/db/db.js

```diff
@@ -84,10 +57,50 @@ async function execute(path, params = {}) {
   sql = sql.replace(/\$\{[^{}]+\}/g, queryParam);
   const values = queryVariables ? queryVariables.map(p => params[p]) : [];
   return query(sql, values);
-}
+};
+
+/**
+ * @description Select all migrations, filter out executed migrations, and executes leftover migrations.
+ * @description Runs a transaction, which will run all leftover migrations, or fail completely.
```

Comment:
you only need the description tag once

```suggestion
 * @description Select all migrations, filter out executed migrations, and executes leftover migrations.
 * Runs a transaction, which will run all leftover migrations, or fail completely.
```

Reply:
Thanks